src/netlink/nl_bridge.cc

```diff
@@ -147,8 +147,16 @@ void nl_bridge::add_interface(rtnl_link *link) {
   // configure bonds and physical ports (non members of bond)
   update_vlans(nullptr, link);
 
-  if (get_vlan_proto() == ETH_P_8021AD)
-    sw->set_egress_tpid(nl->get_port_id(link));
+  auto port_id = nl->get_port_id(link);
+  if (get_vlan_proto() == ETH_P_8021AD) {
+    if (nbi::get_port_type(port_id) == nbi::port_type_lag) {
+      auto members = nl->get_bond_members_by_lag(link);
+      for (auto i : members)
+        sw->set_egress_tpid(i);
+
+    } else
+      sw->set_egress_tpid(port_id);
```

Comment:
Add braces to the else branch as well for readability

```suggestion
    } else {
      sw->set_egress_tpid(port_id);
    }
```